sdk/storage/Azure.Storage.DataMovement/src/Shared/DataMovementExtensions.cs

```diff
@@ -11,42 +11,36 @@ namespace Azure.Storage.DataMovement
 {
     internal static class DataMovementExtensions
     {
-        public static byte[] UriToByteArray(this StorageResource storageResource)
+        // string to byte array
+        public static byte[] ToByteArray(this string query, int? bufferSize = default)
```

Comment:
It's so that if we decide to not do UTF8, I only have to change it one place and not many. Also prevents in the future if someone else has to do this, there's already a set way and not accidentally do ASCII